build.gradle

```diff
@@ -1101,7 +1101,9 @@ ospackage {
 
 buildRpm {
   dependsOn ':packageVSD'
-  dependsOn ':packageZeppelinInterpreter'
+  if (rootProject.hasProperty('enablePublish')) {
+    dependsOn ':packageZeppelinInterpreter'
+  }
   requires('glibc')
```

Comment:
Shouldn't the name be changed for rpm and deb too? I think its "packageName". Since those are only for Linux, so there is no need to add "_linux" to the name.

Reply:
Yes, names are changed for them too. No '_linux' affix for them. Debian package had some restrictions on naming. So the naming is like "tib-compute_1.1.0-1_all.deb" and "tib-compute-1.1.0-1.noarch.rpm"